mavros/launch/px4_config.yaml

```diff
@@ -257,4 +257,8 @@ wheel_odometry:
     frame_id: "odom"
     child_frame_id: "base_link"
 
+# obstacle_distance
+obstacle_distance:
+  mav_frame: "BODY_FRD"       # PX4 1.10 supports GLOBAL, LOCAL_NED, BODY_FRD
```

Comment:
Just FYI, this changes the default, which was global. I think it would be better to keep the default the same, and let people override if they need it.

Reply:
Makes sense, thanks!